src/hotspot/share/gc/parallel/psParallelCompact.hpp

```diff
@@ -367,7 +367,6 @@ class ParallelCompactData
     // These enable optimizations that are only partially implemented.  Use
     // debug builds to prevent the code fragments from breaking.
     HeapWord*            _data_location;
-    HeapWord*            _highest_ref;
 #endif  // #ifdef ASSERT
```

Comment:
Now, we only have one field here. So the word `These` in the comment should be revised to `This`? Or the comment needs more adjustment.

Reply:
Actually, `_data_location` is used only for printing, and it records the corresponding heap-addr of a region, if I read the code correctly. How about removing `_data_location` as well, given its limited value?

Reply:
> it records the corresponding heap-addr of a region, if I read the code correctly.

I searched the code and think you are right. It is good to remove `_data_location`.